cmd/set.go

```diff
@@ -17,15 +18,14 @@ var setCmd = &cobra.Command{
 
     Common statuses: Indexed (default), Idea, Started, Paused, Completed, Aborted, Ongoing, Not Started
     `,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		interactiveFlag, _ := cmd.Flags().GetBool("i") // TODO: Implement this
 		if interactiveFlag {
-			fmt.Println("Not implemented yet")
-			return
+			cmd.SilenceUsage = true
+			return errors.New("Not implemented yet")
 		}
```

Comment:
can you delete these couple lines dealing with the interactive flag please.

I have added support for the interactive command as a sub command and forgot to get rid of this after the implementation was completed.

Reply:
I will let you do it in a separate PR, I'm unsure what you meant

Reply:
sure